docs/SSO.rst

```diff
@@ -1818,11 +1785,6 @@ eXo Platform as SAML2 SP
 			# WARNING: This bundled keystore is only for testing purposes. You should generate and use your own keystore!
 			gatein.sso.picketlink.keystore=${exo.conf.dir}/saml2/jbid_test_keystore.jks
 
-			# Uncomment this when JBoss is used
-
-			#gatein.sso.login.module.class=org.gatein.sso.agent.login.SAML2WildflyIntegrationLoginModule
-			#gatein.sso.uri.suffix=dologin
-
 			# Uncomment this when Tomcat is used
```

Comment:
this should be removed too since tomcat is the only option. And it should be reflected in .properties if properties already exist in default packaing